cpp/tests/CMakeLists.txt

```diff
@@ -327,51 +327,51 @@ ConfigureTest(MST_TEST tree/mst_test.cu)
 # - Experimental Graph tests ----------------------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_GRAPH_TEST experimental/graph_test.cpp)
+ConfigureTest(GRAPH_TEST structure/graph_test.cpp)
 
 ###################################################################################################
 # - Experimental weight-sum tests -----------------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_WEIGHT_SUM_TEST experimental/weight_sum_test.cpp)
+ConfigureTest(WEIGHT_SUM_TEST structure/weight_sum_test.cpp)
 
 ###################################################################################################
 # - Experimental degree tests ---------------------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_DEGREE_TEST experimental/degree_test.cpp)
+ConfigureTest(DEGREE_TEST structure/degree_test.cpp)
 
 ###################################################################################################
 # - Experimental coarsening tests -----------------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_COARSEN_GRAPH_TEST experimental/coarsen_graph_test.cpp)
+ConfigureTest(COARSEN_GRAPH_TEST structure/coarsen_graph_test.cpp)
 
 ###################################################################################################
 # - Experimental induced subgraph tests -----------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_INDUCED_SUBGRAPH_TEST experimental/induced_subgraph_test.cpp)
+ConfigureTest(INDUCED_SUBGRAPH_TEST community/induced_subgraph_test.cpp)
 
 ###################################################################################################
 # - Experimental BFS tests ------------------------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_BFS_TEST experimental/bfs_test.cpp)
+ConfigureTest(BFS_TEST traversal/bfs_test.cpp)
 
 ###################################################################################################
 # - Experimental Multi-source BFS tests -----------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_MSBFS_TEST experimental/ms_bfs_test.cpp)
+ConfigureTest(MSBFS_TEST traversal/ms_bfs_test.cpp)
 
 ###################################################################################################
 # - Experimental SSSP tests -----------------------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_SSSP_TEST experimental/sssp_test.cpp)
+ConfigureTest(SSSP_TEST traversal/sssp_test.cpp)
 
 ###################################################################################################
 # - Experimental PAGERANK tests -------------------------------------------------------------------
-ConfigureTest(EXPERIMENTAL_PAGERANK_TEST experimental/pagerank_test.cpp)
+ConfigureTest(PAGERANK_TEST pagerank/pagerank_test.cpp)
```

Comment:
To be consistent with python, shouldn't this better go to link_analysis (after creating the directory)?

Reply:
Except I disagree with that python choice :-)

I'll make that change.

Reply:
The `src` directory also uses `link_analysis`.  Now src, tests and python all match.